nodes/src/nodes/Sql.Query.BulkInsert.ts

```diff
@@ -0,0 +1,56 @@
+import { ModuleCompute, ModuleDefinition } from '@nodescript/core/types';
+
+import { BulkInsertBuilder, BulkInsertParams, BulkInsertResult } from '../lib/BulkInsertBuilder.js';
+
+export const module: ModuleDefinition<BulkInsertParams, BulkInsertResult> = {
+    version: '0.0.1',
+    moduleName: 'SQL / Query / Bulk Insert',
+    description: 'Converts an input array representing rows of data to a SQL INSERT query string and params array.',
+    keywords: ['sql', 'database', 'insert', 'bulk', 'query'],
+    params: {
+        vendor: {
+            schema: { type: 'string', enum: ['PostgreSQL', 'MySQL'] },
+        },
+        tableName: {
+            schema: { type: 'string' }
+        },
+        rowData: {
+            schema: {
+                type: 'array',
+                items: {
+                    type: 'object',
+                    properties: {},
+                    additionalProperties: { type: 'any' }
+                },
+                description: 'Array of objects representing row data for insertion'
+            }
+        },
+        returning: {
```

Comment:
I reckon we shouldn't do this as part of this node, so let's probably remove (and from builder too). I reckon anything vendor-specific should be composed manually (at least in the initial phase, until we have more sophisticated builders).

Reply:
ah yeah sorry you did say this previously - this was left in by mistake